packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx

```diff
@@ -84,5 +94,11 @@ export const PhaseProgress = ({ currentPhase }: PhaseProgress) => (
             <Translation id={COINJOIN_PHASE_MESSAGES[currentPhase]} />
             ...
         </Message>
+
+        {phaseDeadline && (
```

Comment:
The condition seems redundant.

Reply:
dcd11dc6f606fcbde9a674b4050ce92b690ab9c3